runhouse/resources/function.py

```diff
@@ -87,11 +89,17 @@ def to(
             >>> rh.function(fn=local_fn).to(gpu_cluster)
             >>> rh.function(fn=local_fn).to(system=gpu_cluster, env=my_conda_env)
         """
+        from runhouse.resources.serverless import aws_lambda_function
```

Comment:
move inside the if statement where it would be called?

Reply:
makes sense, moved.